### PR TITLE
I-DOCSIS 6.1.9: 37134+37135

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/EPM/I-DOCSIS/EPM_6.1.9_I-DOCSIS.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#### US QAM Ch Bitrate and DS QAM Ch Bitrate trending added for CCAP connectors [ID_37134]
+
+​The following parameters have been added to the trend template for the CCAP connectors:
+
+- QAM US Channels: US QAM Ch Bitrate
+- QAM DS Channels: DS QAM Ch Bitrate
+
+#### New OFDM/OFDMA information available [ID_37202]
+
+In the CISCO CBR-8 CCAP Platform connector, a new *OFDM/OFDMA* page has been added containing parameters related to channels from the device. These channels are also linked to the associated node segments and service groups. When you view either of those entities in the topology, new KPIs are displayed with the total number of OFDM and OFDMA channels and the average utilization. There is also a link to a dashboard displaying all channels associated with the entity.
```
Comment (MariekeGO) [inline release-notes/EPM/I-DOCSIS/EPM_6.1.9_I-DOCSIS.md:21]:
```suggestion
In the CISCO CBR-8 CCAP Platform connector, a new *OFDM/OFDMA* page has been added containing parameters related to OFDM/OFDMA channels from the device. Currently this information is only available at node segment level. These channels are also linked to the associated node segments and service groups. When you view either of those entities in the topology, new KPIs are displayed with the total number of OFDM and OFDMA channels and the average utilization. There is also a link to a dashboard displaying all channels associated with the entity.
```

File: release-notes/EPM/I-DOCSIS/EPM_6.1.9_I-DOCSIS.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#### Creating new CCAP/CM pairs from a CSV file [ID_37262]
+
+It is now possible to create CCAP/CM pairs based on a CSV file.
+
+The existing script *EPM_I_DOCSIS_AddNewCcapCmPair* has been expanded with the possibility to specify the path to a CSV file and then create the CCAP/CM pairs based on that file. The CSV file must have the following structure: *ElementName, Be_DMA, Collector_DMA, Protocol , IpAddress, CommunityString, Network, Market, Hub, SystemUser, SystemPass*.
+
+#### Topology chains can now be hidden [ID_37516]
+
+The Skyline EPM Platform connector has been updated so that you can now hide entire topology chains or specific fields such as DOCSIS Market, CM, CCAP Core, etc. To access this feature, navigate to the Chain and Fields selection page of the EPM front-end element. There you can toggle the visibility of fields reflected in the topology chain in the sidebar on the left.
+
```
[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/EPM/I-DOCSIS/EPM_6.1.9_I-DOCSIS.md:40]: Do you mean that I should add that this is only possible when using the local administrator? Or does this still need to be verified?

[the remark this answers is not in the document]
Comment (miguel-slc) [reply release-notes/EPM/I-DOCSIS/EPM_6.1.9_I-DOCSIS.md:40]: Do you mean that I should add that this is only possible when using the local administrator? 
Yes.

Background:
EPM elements have the data display pages hidden by default (for performance reasons). Only the local administrator can see it. We surpass this restriction by making these parameters available in the Visio file assigned to the FE element. In that way any user can see these parameters. The Visio file was not created yet, which means that only the local administrator can see these settings

**Comment by MariekeGO** [comment]
@miguel-slc, I've implemented the necessary changes based on your feedback. Could you check if everything looks OK now?

**Pull request merged**